package.json

```diff
@@ -22,6 +22,7 @@
   "license": "MIT",
   "dependencies": {},
   "devDependencies": {
+    "sigmund": "^1.0.1",
```

Comment:
This is specifically because it's used in a test, I wanted to make sure it was clear that it's something the test setup is relying on.